src/components/ToDo.js

```diff
@@ -25,38 +26,78 @@ const mainTodos = [
 const ToDo = () => {
   const [todos, setTodos] = useState(mainTodos);
   const [inputText, setInputText] = useState("");
+  const [alert, setAlert] = useState({ show: false });
```

Comment:
```suggestion
  const [showAlert, setShowAlert] = useState(false);
```
It should be simpler. No need to map object.